includes/news/v2/mail/featured-stories.php

```diff
@@ -7,7 +7,7 @@
 
 	foreach( $featured_stories as $index => $details ) {
 		extract( $details );
-		if( $$index == $limit ) break;
+		if( $index == $limit ) break;
```

Comment:
Should this be a strict equality, or is there a change $index and $limit are different types?

Reply:
The $limit is defined above as an int and the $index should always be an int